.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Android-ci
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: docker://fabernovel/android:api-29-v1.1.0
```

Comment:
Can you try using circleci image?

Reply:
@jlin-sift Per Github [actions documentation](https://github.com/marketplace/actions/docker-android-action) above image was suggested for the android build.

Reply:
Based on https://github.com/SiftScience/sift-android/blob/master/sift/build.gradle#L15-L16,  SdkVersion is 31, and buildToolsVersion is 30.0.0. It does not look like `fabernovel/android:api-29-v1.1.0` has these versions. See https://hub.docker.com/layers/fabernovel/android/api-29-v1.1.0/images/sha256-10f45f7c3a75f7220d58bf24039f3f7ac492c42dd0ebf37f76fa6010fe595271?context=explore.  You might want to try `api-31-v1.6.0`.

Reply:
Sure @jlin-sift Since it was the same version used in circleCI pipelineconfig i have added api-29-v1.1.0.
Let me update it to `api-31-v1.6.0`.

Reply:
@jlin-sift Latest build after above suggested update.
https://github.com/SiftScience/sift-android/actions/runs/9669426792/job/26675896118